lib/reporter.rb

```diff
@@ -0,0 +1,37 @@
+require 'active_support'
+require 'active_support/core_ext'
+require 'logger'
+
+require_relative './reporter/alb_errors_report_builder'
+require_relative './reporter/authentications_report_builder'
+require_relative './reporter/doc_auth_dropoff_report_builder'
+
+module Reporter
+  def self.start_reporting
+    Thread.new do
+      Time.zone = ActiveSupport::TimeZone.new('Eastern Time (US & Canada)')
```

Comment:
What if we just set a constant somewhere on startup that was like `TIME_ZONE`? The `Thread.new` makes me worry about races

Reply:
Good call, I'll do that